contrib/local-environment/docker-compose-keycloak.yaml

```diff
@@ -43,7 +43,7 @@ services:
 
   keycloak:
     container_name: keycloak
-    image: keycloak/keycloak:25.0
+    image: keycloak/keycloak:26.0
```

Comment:
have you tested that the realm and user files still work?

Reply:
I thought that these aspects were tested in the project pipeline automatically.
I saw that the pipeline was wholly green, so I assumed there would be no problems with the change.
The same applies below.

Reply:
Currently none of these files are used in any automated tests. They currently only serve as examples and for manual testing purposes.

That's why I keep the renovate PRs open for so long because it's not a priority for me to test them all.

Furthermore, we are currently working on a new e2e-testing repository for actually doing more then just unit testing and I would like to move all the files in the contrib directory into it in the next few weeks.

Therefore if it is fine with you I will not merge this PR for now?

Reply:
As you prefer.